spec/controllers/annotation_categories_controller_spec.rb

```diff
@@ -187,6 +224,37 @@
 
       expect(text.reload.content).to eq 'updated content'
     end
+
+    it 'successfully updates an annotation text\'s (associated with an annotation category) deduction' do
+      assignment_w_deductions = create(:assignment_with_deductive_annotations)
+      category = assignment_w_deductions.annotation_categories.where.not(flexible_criterion_id: nil).first
+      text = category.annotation_texts.first
+      put :update_annotation_text,
+          params: {
+            assignment_id: category.assessment_id,
+            id: text.id,
+            annotation_text: { content: 'more updated content', deduction: 0.1 },
+            format: :js
+          }
+
+      expect(text.reload.deduction).to eq(0.1)
```

Comment:
As a stylistic thing, don't put parentheses around the argument to `eq`. So `eq 0.1` instead of `eq(0.1)`.

Reply:
Oh no! I've been doing this for a while